include/IECore/InternedString.inl

```diff
@@ -116,6 +116,11 @@ inline const char *InternedString::c_str() const
 	return m_value->c_str();
 }
 
+inline size_t tbb_hasher( const InternedString &s )
+{
+	return std::hash<std::string>()( s.string() );
+}
+
```

Comment:
I've just discovered that this conflicts with https://github.com/GafferHQ/gaffer/blob/master/src/GafferSceneTest/ContextSanitiser.cpp#L56-L60. This is OK for GafferHQ as we can remove the duplicate on master at the point we upgrade dependencies, but are you going to need an #ifdef for builds at IE?

Reply:
Yes please

Reply:
Added #define in 98ecd8d, and opened Gaffer PR to use it in GafferHQ/gaffer#3956.